apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts

```diff
@@ -243,6 +246,23 @@ export class BookingsController {
     throw new InternalServerErrorException("Could not create instant booking.");
   }
 
+  @Post("/no-show")
```

Comment:
I think `/:bookingUid/no-show` makes more sense. As we are making an update to the Booking resource and not all Bookings resource

Reply:
This endpoint also updates `Attendee`  table noShow field.

Reply:
Yeah, I still think it is better because Attendee is connected to booking.


Though there could be a separate endpoint in future to work with attendee directly if required, that would also allow no-show update.

Reply:
Yeah agreed with @hariombalhara. You shouldn’t be able to directly update attendees without going through the booking